tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -118,6 +118,20 @@ trait WritableVirtualJSFile extends WritableVirtualTextFile with VirtualJSFile {
   def sourceMapWriter: Writer
 }
 
+/** An virtual file containing Scala.js IR.
+ *
+ *  This can be a VirtualScalaJSIRFile (with RelativeVirtualFile) or a
+ *  VirtualFileContainer.
+ *
+ *  The main difference to using individual files
```

Comment:
Is my English too weak, or should this be "The main difference <ins>compared</ins> to using ..."?